script/scrape_config.sh

```diff
@@ -73,6 +82,14 @@ fi
 #rm /tmp/sara_install_ws
 
 
+######################################
+## Set back to https if needed
+if [[ $FORCE_SSH == "true" ]]
```

Comment:
Every https seems to have the wrong $FORCE_SSH value

Reply:
voir ma réponse au dessu